buildkite/src/Jobs/ClientSdk.dhall

```diff
@@ -23,14 +24,34 @@ Pipeline.build
         name = "ClientSdk"
       },
     steps = [
-    Command.build
-      Command.Config::{
-        commands = OpamInit.andThenRunInDocker ([] : List Text) "./buildkite/scripts/build-client-sdk.sh",
-        label = "Build client-sdk",
-        key = "build-client-sdk",
-        target = Size.Medium,
-        docker = None Docker.Type
-      }
+      Command.build
+        Command.Config::{
+          commands = OpamInit.andThenRunInDocker ([] : List Text) "./scripts/client_sdk_unit_tests.sh",
+          label = "Client SDK unit-tests",
+          key = "client-sdk-unittests",
+          target = Size.Medium,
+          docker = None Docker.Type
+        },
+      Command.build
+        Command.Config::{
+          commands = OpamInit.andThenRunInDocker ([] : List Text) "cd frontend/client_sdk && yarn prepublishOnly",
+          label = "Build And Test Client SDK",
```

Comment:
Should this step have a different label? The actual build occurs in the previous step.

Reply:
Sure tho basing it off: https://github.com/CodaProtocol/coda/pull/5742/files#diff-1d37e48f9ceff6d8030570cd36286a61R368